File: 프로그래머스/1/86051. 없는 숫자 더하기/없는 숫자 더하기.py
def solution(numbers):
    
    return 45 - sum(numbers)

# 다른 풀이
return sum([i for i in [1,2,3,4,5,6,7,8,9,0] if i not in numbers])

# 다른 풀이
answer=0
for i in range(1,10):
    if i not in numbers:
        answer += i
return answer

# 다른 풀이
return sum(set(range(10)) - set(numbers))

File: 프로그래머스/1/86051. 없는 숫자 더하기/없는 숫자 더하기.java
class Solution {
    public int solution(int[] numbers) {
        int answer = 0;
        int max = 0;
        
        for(int i = 0 ; i <= 9; i++){
            max += i;
            for(int num : numbers) {
                if(num == i) {
                    answer += i;
                    break;
                } 
            }
        }
        
        return max - answer;
    }
}